tensorflow/stream_executor/kernel.h

```diff
@@ -349,6 +349,7 @@ class TypedKernel : public KernelBase {
         port::InlinedVector<uint8, 4>{arg_ptr, arg_ptr + sizeof(arg)}, sizeof(arg)});
   }
 
+#ifndef GOOGLE_CUDA
```

Comment:
why is this required?  this is suspicious -- can you give us the output of compiling when this happens?